<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HP" name="Task 5 [Android]" tests="2" failures="1" timestamp="18 Oct 2017 18:00:25 GMT" time="10.866" errors="0">
  <testcase name="checkSiteVersion" time="9.378" classname="task5.QATestTask5.tests.PlaceOrderTest"/>
  <testcase name="createNewOrder" time="1.464" classname="task5.QATestTask5.tests.PlaceOrderTest">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 7, Size: 7">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 7, Size: 7
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at task5.QATestTask5.tests.PlaceOrderTest.clickRandomProduct(PlaceOrderTest.java:116)
	at task5.QATestTask5.tests.PlaceOrderTest.createNewOrder(PlaceOrderTest.java:93)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- createNewOrder -->
</testsuite> <!-- Task 5 [Android] -->
